arm/runui.py

```diff
@@ -5,6 +5,22 @@
 from arm.config.config import cfg  # noqa E402
 import arm.ui.routes  # noqa E402
 
+host = cfg['WEBSERVER_IP']
+if host == 'x.x.x.x':
```

Comment:
As the default is `x.x.x.x`, and to make this more resilient to dynamic IP changes, allow user to leave the value as `x.x.x.x` and try to auto-detect the IP address (preferring to use an address that is not localhost)

Reply:
interested to see how this works.